Source/Libraries/openXDA.Model/System/DataOperationFailure.cs

```diff
@@ -35,7 +35,6 @@ public class DataOperationFailure
         [PrimaryKey(true)]
         public int ID { get; set; }
 
-        [ParentKey(typeof(DataOperation))]
```

Comment:
Why is this being removed? Is this field not a foreign key that references the `DataOperation` table?

https://github.com/GridProtectionAlliance/openXDA/blob/c53687fe0491e261c3cf6eb399e58d9ff6b86819/Source/Data/01%20-%20openXDA.sql#L236

Reply:
Having more than one parent key on a model causes issues because there is an implicit assumption that there is only one, and the datafile parent key is more valuable.

Reply:
I'll try harder to remember that. I think I've asked this before.